src/components/organisms/AdminVenues/AdminVenues.tsx

```diff
@@ -16,35 +17,39 @@ export interface AdminVenuesProps {
 
 export const AdminVenues: React.FC<AdminVenuesProps> = ({ venues }) => {
   const partyVenues = useMemo(
-    () => venues?.filter((venue) => venue.template === VenueTemplate.partymap),
+    () =>
+      venues
+        ?.filter((venue) => venue.template === VenueTemplate.partymap)
```

Comment:
`src/types/venues.ts:291`

```suggestion
        ?.filter(isPartyMapVenue)
```

Does this actually care if it's a Party Map venue specifically? Or only if it's a venue with rooms?

```suggestion
        ?.filter(isVenueWithRooms)
```

Reply:
These should be partymap venues only. Because admin v2 creates a party map by default and all of the cards should be party map venues only. This is probably going to change after we add the worlds concept.

Reply:
Sounds good.